test/execute-functional-tests-cloud.sh

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env bash
+
+# ------------------------------------------------------------
+# Copyright 2023 The Radius Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ------------------------------------------------------------
+
+# Make sure that you have default environment in the default group.
+# The way to check that is to run the following commands:
+# `rad group switch default`
+# `rad env list` and check if there is an environment named `default`.
```

Comment:
If this is expected to be already existing, then the manual group creation on L30 shouldn't be needed